### DISCUSSION TITLE
Is there an approximate release time for the mobile alpha version?

### DISCUSSION THREAD

**Discussion by zwl1619** [tauri-apps/tauri#5568] [Q&A] [answered]
Is there an approximate release time for the mobile alpha version?

**Comment by FabianLars** [comment]
Shortly after 1.2 (no exact ETA, but pretty soon)

**Comment by nothingismagick** [comment, marked as answer]
Today.
See this merge:
https://github.com/tauri-apps/tauri/pull/5787/commits/df2e027dd97b9cab7cbe350b40bde9bea31d05fe
